CMakeLists.txt

```diff
@@ -193,7 +193,10 @@ else ()
   option(USE_DEBUG_MALLOC "Build oeenclave with memory leak detection capability." OFF)
 endif ()
 
-option(ADD_WINDOWS_ENCLAVE_TESTS "Build Windows enclave tests" OFF)
```

Comment:
Why this is line is removed?

Reply:
Because rebasing through dozens of conflicts is error prone. I will fix that, sorry.

Reply:
Thanks!

Reply:
Thanks for catching this, fixed now.